app/tests/integration/e2e_test_variables.json

```diff
@@ -24,5 +24,53 @@
                 "firefox_pref_branch": "default"
             }
         ]
+    },
+    "multi-pref-experiment": {
+        "action_name": "multi-preference-experiment",
+        "name": "E2E experimenter multipref-test",
+        "type": "channel",
+        "channels": "Nightly",
+        "min_version": "80.0",
+        "max_version": "100.0",
+        "userFacingName": "e2e testing name",
+        "userFacingDescription": "e2e testing description",
+        "branches": [
+            {
+                "branch_description": "multipref-branch-description",
+                "branch_name": "e2e-multipref-branch",
+                "preferences": [
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-1",
+                        "firefox_pref_type": "boolean",
+                        "firefox_pref_value": "true"
+                    },
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-1-1",
+                        "firefox_pref_type": "integer",
+                        "firefox_pref_value": "10"
+                    }
+                ]
+            },
+            {
+                "branch_description": "multipref-branch-description-2",
+                "branch_name": "e2e-multipref-branch-2",
+                "preferences": [
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-2",
+                        "firefox_pref_type": "string",
+                        "firefox_pref_value": "pref-string"
+                    },
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-2-1",
+                        "firefox_pref_type": "string",
```

Comment:
This should be: `"firefox_pref_type": "json string",`

Defined here:

https://github.com/mozilla/experimenter/blob/master/app/experimenter/experiments/constants.py#L226

Reply:
This is what I mentioned in Slack. Setting the test variable as ```json string``` results in a failure because the ```/recipe``` api changes the json string selection to just ```string```.

Reply:
@jrbenny35 Yeah it has to be `"json string"` going into the input and `"string"` coming out in the recipe, but we need to do both.